docs/wallets/helium-wallet-app.mdx

```diff
@@ -173,12 +177,12 @@ the Helium blockchain.
   src={useBaseUrl('img/wallets/helium-wallet-app/address-book-banner.png')}
 />
 
-This feature allows you to save names/nicknames and related HNT wallet addresses for those you may
-wish to send HNT to. Contacts are displayed in alphabetical order, and are searchable by name or
-wallet address. There is no limit to the number of Contacts you can have.
+This feature allows you to save names/nicknames and related Helium or Solana allet addresses for 
```

Comment:
```suggestion
This feature allows you to save names/nicknames and related wallet addresses for 
```
Prefer to not draw to much contrast between Helium/Solana accounts since they're one in the same.